scripts/wikipedia_covid/timeseries/CovidWikipedia.mcf

```diff
@@ -0,0 +1,7 @@
+Node: dcid:Count_MedicalConditionIncident_COVID_19_ConfirmedCase
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalConditionIncident
+statType: dcs:measuredValue
+measuredProperty: dcs:count
+medicalStatus: dcs:ConfirmedCase
+medicalCondition: dcs:COVID_19
```

Comment:
Same note as from other PR re incidentType. BTW these are the same StatVar? If one is submitted first, we can remove the other.

Reply:
sounds good